core/src/datayoga_core/blocks/relational/utils.py

```diff
@@ -39,15 +40,29 @@ def get_engine(connection_name: str, context: Context, autocommit: bool = True)
         lib_dir = connection.get("oracle_thick_mode_lib_dir")
         extra["thick_mode"] = {'lib_dir': lib_dir} if lib_dir else {}
 
+    # PSQL specific parameters to be passed in the query string for SSL connection
+    if db_type == DbType.PSQL:
+        args = connection.get("connect_args", {})
+        ssl_args["sslmode"] = args.get("sslmode")
+        ssl_args["sslrootcert"] = args.get("sslrootcert")
+        ssl_args["sslkey"] = args.get("sslkey")
+        ssl_args["sslcert"] = args.get("sslcert")
+        ssl_args["sslpassword"] = args.get("sslpassword")
```

Comment:
you can implement with a loop on the sslmode, sslrootcert, etc:
```
for prop in ["sslmode", "sslrootcert" ..]:
   ssl_args[prop] = args.get(prop)
``

Reply:
✅